src/components/ApplicationDashboard/ApplicationDashboard.tsx

```diff
@@ -1,11 +1,158 @@
 import React, { FC } from "react";
-import { RouteComponentProps } from "@reach/router";
 
-interface Props extends RouteComponentProps {}
+import Link from "@govuk-react/link";
+import { RouteComponentProps, Link as RouterLink } from "@reach/router";
+
+import { Title } from "ukripoc-components";
+
+import { H4 } from "@govuk-react/heading";
+import { NTA_LIGHT } from "@govuk-react/constants";
+import { GREY_3, GREY_2 } from "govuk-colours";
+
+import GridRow from "@govuk-react/grid-row";
+import GridCol from "@govuk-react/grid-col";
+
+import styled from "styled-components";
+import { ListFundingApplicationsQuery } from "../../API";
+
+export const ApplicationContainer = styled.div`
+    background: ${GREY_3};
+    padding: 20px;
+    margin-bottom: 30px;
+`;
+
+export const ApplicationContainerItem = styled.div`
+    font-family: ${NTA_LIGHT};
+    background: white;
+    padding: 20px;
+    margin-top: 10px;
+    border: 1px solid ${GREY_2};
+`;
+
+export const ApplicationContainerItemComplete = styled.div`
+    font-family: ${NTA_LIGHT};
+    font-weight: bold;
+    line-height: 70px;
+    text-align: center;
+    border-left: 1px dotted ${GREY_3};
+`;
+
+export const ApplicationContainerTimeline = styled.div`
+    font-family: ${NTA_LIGHT};
+    text-align: center;
+`;
+
+interface Props extends RouteComponentProps {
+    applications?: ListFundingApplicationsQuery; //change when not mocked
+}
 
 export const ApplicationDashboard: FC<Props> = props => {
-    console.log(props);
-    return <div>Dashboard</div>;
+    const applications =
+        props.applications &&
+        props.applications.listFundingApplications &&
+        props.applications.listFundingApplications.items;
+
+    function daysLeft(date: string | null) {
```

Comment:
Extract this outside the component into a helper that generates the time and suffix (returning the strings, not the JSX. You could do this as an array which you imemdiately destructure). A general rule is to not define functions inside the component unless they need access to hook data. This